src/coreComponents/constitutive/fluid/CompositionalMultiphaseFluid.cpp

```diff
@@ -38,13 +38,13 @@ namespace
 
 PVTPackage::EOS_TYPE getCompositionalEosType( string const & name )
 {
-  static std::map< string, PVTPackage::EOS_TYPE > const eosTypes =
+  static std::map< string, PVTPackage::EOS_TYPE > const EOS_TYPES =
```

Comment:
why ALL_CAPS?

Reply:
`static const` are often upper case. We can remove if needed.